### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by chanchal16**
- Created a basic video listing page

https://user-images.githubusercontent.com/56486295/163601920-3c7ee41d-1c29-43cd-b777-23275cee379c.mp4

### CODE REVIEW

File: src/pages/VideoListing.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export function VideoListing() {
+    const{videos,setVideos} = useVideo();
+
+    useEffect(() => {      
+      (async () => {
+        try {
+           await axios.get("/api/videos")
+           .then((res) => {
+                setVideos(res.data.videos);
+                console.log('videos',res.data.videos)
```
Comment (sonishreyas) [inline src/pages/VideoListing.jsx:17]: Just incase you missed it

**Review by sonishreyas — APPROVED** [review]
Code looks good, really like the design

File: src/pages/VideoListing.jsx (hunk 1 above)
Comment (chanchal16) [reply src/pages/VideoListing.jsx:17]: Ohh yes, I missed it will remove it.

File: src/components/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    <span class=" gray text-xs">{video.views}views</span>                                         
+                </div>
+                <div className='action-btn'>
+                    <span className='gray'><MdPlaylistAdd size='1.3rem'/></span>
+                    <span className='gray'><MdOutlineWatchLater size='1.2rem'/></span>
+                </div>
+            </div>
+        </div>
+    </div>
+  )
```
Comment (Sharmaryan) [inline src/components/VideoCard.jsx:29]: destructure from 'video' , will be good for readability

**Review by Sharmaryan — APPROVED** [review]
suggested a small change only, everything is fine, also UI is looking nice

**Pull request merged**